roles/core/conman/tasks/main.yml

```diff
@@ -30,23 +30,66 @@
   tags:
     - package
 
+- name: group █ Ensure group conman exists
+  group:
+    name: conman
+    state: present
+    system: yes
+
+- name: user █ Ensure user conman exists
+  user:
+    name: conman
+    state: present
+    comment: "Connection Manager service"
+    group: conman
+    home: /var/run/conman
+    shell: /bin/false
+    system: yes
+
 - name: file █ Create /var/log/conman directory
   file: # noqa 208
     path: /var/log/conman
-    setype: conman_log_t
+    setype: "{{ (ansible_facts.selinux.status == 'enabled') | ternary('conman_log_t', omit) }}"
     state: directory
+    owner: conman
+    group: conman
     recurse: yes
+  notify: service █ Restart conman
+  when:
+    - ansible_facts.os_family == "RedHat"
+
+- name: file █ Ensure systemd drop-in directory exists
+  file:
+    path: /etc/systemd/system/conman.service.d
+    mode: 0755
+    state: directory
+    owner: root
+    group: root
+  when:
+    - ansible_facts.os_family == "RedHat"
+
+- name: copy █ Ensure service run with user conman
+  copy:
+    content: |
+      [Service]
+      User=conman
+      Group=conman
+      ExecStart=
+      ExecStart=/usr/sbin/conmand -P /var/run/conman/conmand.pid
```

Comment:
Why 2 ExecStart with first one empty ?

Reply:
This is explained in the commit message:  Override ExecStart with a new path for PIDFile (need to clear ExecStart first) owned by the conman user, this to allow the command to write the PIDFile.

ExecStart can be defined several time. To clear any previous value, we need to define it with an empty value.